insurance_cloudevents/README.md

```diff
@@ -30,29 +30,32 @@
    - Use sklearn framework  
 3. Click Submit.
 4. Open Jupyterlab and from **workspace/insurance/insurance_cloudevents** open [resources.ipynb](https://github.com/oneconvergence/dkube-examples/tree/monitoring/insurance_cloudevents/resources.ipynb) and fill the following details in the first cell.
+     - **DKUBE_IP** = {IP of the DKube setup without https:// and without port number eg: 1.2.3.4}
      - **DKUBEUSERNAME** = {your dkube username}
      - **MODELMONITOR_NAME** = {your model monitor name}
-     - **S3_KEY** = {MINIO access key}
-     - **S3_SECRET_KEY** = {MINIO access secret key}
-     - **S3_ENDPOINT** = {MINIO Endpoint}
```

Comment:
we need the endpoint, right?

Reply:
Like I mentioned http://<ip>:32221 where ip is the DKube ip address where the deployment is running